src/wallet/feebumper.cpp

```diff
@@ -254,6 +252,7 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
             txin.scriptSig.clear();
             txin.scriptWitness.SetNull();
         }
+        mtx.vout = txouts;
```

Comment:
while you are here, could rename `mtx` so it doesn't shadow the function's arg (or rename the argument).

Reply:
Changed to `temp_mtx`